core/src/main/scala/com/microsoft/azure/synapse/ml/logging/common/SparkHadoopUtils.scala

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.logging.common
+
+import org.apache.spark.SparkContext
+
+import com.microsoft.azure.synapse.ml.logging.SynapseMLLogging
+
+object SparkHadoopUtils {
+
+  def getHadoopConfig(key: String, SC: SparkContext): String = {
```

Comment:
SC -> sc and it shouldnt be null

Reply:
You are right, all the jobs are running on Spark. And that's were these usage utils are called from. So, SparkContext should never be null. I will remove the check.